keyboards/keyhive/maypad/keymaps/readme.md

```diff
@@ -0,0 +1 @@
+# The default keymap for maypad
```

Comment:
This file should be in the same subdirectory as the companion `keymap.c`.

Reply:
this as well is fixed in an earlier commit